src/SelectMenu/SelectMenu.js

```diff
@@ -51,8 +51,9 @@ const StyledSelectMenu = styled.details`
 `
 
 // 'as' is spread out because we don't want users to be able to change the tag.
-const SelectMenu = ({children, initialTab, as, ...rest}) => {
-  const ref = useRef(null)
+const SelectMenu = React.forwardRef(({children, initialTab, as, ...rest}, forwardedRef) => {
+  const backupRef = useRef()
```

Comment:
Based on https://reactjs.org/docs/forwarding-refs.html#note-for-component-library-maintainers, could we give this function a name so it shows up as expected in devtools?

```suggestion
const SelectMenu = React.forwardRef(
  function SelectMenu({children, initialTab, as, ...rest}, forwardedRef) {
    const backupRef = useRef()
```

Reply:
The name seems to show up fine in devTools! 
![image](https://user-images.githubusercontent.com/8960591/95139973-92e9a400-0722-11eb-8eea-5067bc4c0714.png)